node/node_test.go

```diff
@@ -390,6 +390,73 @@ func TestNode_GetAllESDTTokens(t *testing.T) {
 	assert.Equal(t, esdtData, value[esdtToken])
 }
 
+func TestNode_GetAllESDTTokensShouldReturnEsdtAndFormattedNft(t *testing.T) {
+	acc, _ := state.NewUserAccount([]byte("newaddress"))
```

Comment:
t.Parallel ?

Reply:
no test in this file contains `t.Parallel()` so I would leave it as it is